tests/ci/docker_images/linux-x86/ubuntu-22.04_gcc-12x/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu-22.04:base
+
+SHELL ["/bin/bash", "-c"]
+
+# gcc-11 is relatively new and not readily available, so we must fetch it from a mirror for now.
+RUN set -ex && \
+    # echo "deb http://mirrors.kernel.org/ubuntu hirsute main universe" >> /etc/apt/sources.list && \
```

Comment:
This and the comment on Line 8 should only apply to gcc-11. 
But It looks like I can update the gcc-11 docker image with the new `ubuntu-22.04_base` docker image now, instead using my previous workaround :)

Reply:
Removed.